ads/google/a4a/test/test-utils.js

```diff
@@ -231,6 +232,38 @@ describe('Google A4A utils', () => {
     });
   });
 
+  describe('#getAmpRuntimeTypeParameter', () => {
+    it('should specify that this is canary', () => {
+      expect(getAmpRuntimeTypeParameter({
+        AMP_CONFIG: {type: 'canary'},
+        location: {href: 'https://www-pub-com.cdn.ampproject.org'},
+      })).to.equal('2');
+    });
+    it('should specify that this is control', () => {
+      expect(getAmpRuntimeTypeParameter({
+        AMP_CONFIG: {type: 'control'},
+        location: {href: 'https://www-pub-com.cdn.ampproject.org'},
+      })).to.equal('1');
+    });
+    it('should not have `art` parameter when AMP_CONFIG is undefined', () => {
+      expect(getAmpRuntimeTypeParameter({
+        location: {href: 'https://www-pub-com.cdn.ampproject.org'},
+      })).to.be.null;
+    });
+    it('should not have `art` parameter when binary type is production', () => {
+      expect(getAmpRuntimeTypeParameter({
+        AMP_CONFIG: {type: 'production'},
+        location: {href: 'https://www-pub-com.cdn.ampproject.org'},
+      })).to.be.null;
+    });
+    it('should not have `art` parameter when canonical', () => {
+      expect(getAmpRuntimeTypeParameter({
+        AMP_CONFIG: {type: 'canary'},
+        location: {href: 'https://www.pub.com'},
+      })).to.be.null;
+    });
+  });
+
```

Comment:
Using `example.com` would be better than `pub.com` if we were starting from scratch, since `pub.com` is a real site.  But not worth changing now, at least not as part of this PR.

Reply:
Fixed.